variants/WRAITH32_F051K6/variant.h

```diff
@@ -0,0 +1,116 @@
+/*
+  Copyright (c) 2011 Arduino.  All right reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+#define PA10 0
+#define PA9  1
+#define PA12 2
+#define PB0  3
+#define PB7  4
+#define PB6  5
+#define PB1  6
+#define PF0  7
+#define PF1  8
+#define PA8  9
+#define PA11 10
+#define PB5  11
+#define PB4  12 // LED_BLUE
+#define PB3  13 // LED_GREEN
+#define PA0  14 // A0
+#define PA1  15 // A1
+#define PA2  16 // A2 - RC Input
+#define PA3  17 // A3
+#define PA4  18 // A4
+#define PA5  19 // A5 
+#define PA6  20 // A6
+#define PA7  21 // A7
+#define PA15 22 // LED_RED
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        23
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       8
+#define NUM_ANALOG_FIRST        14
+
+// On-board LED pin number
+#define LED_BUILTIN             13
```

Comment:
I advise to use PYn notation, more readable and also avoid misalignment  if pins number are changed.

Reply:
Done